portlets/src/main/webapp/vue-app/realizations/realizationsServices.js

```diff
@@ -27,8 +27,8 @@ export function updateRealization( id, status, actionLabel, domain ,points) {
   });
 }
 
-export function exportFile(fromDate, toDate) {
-  window.open(`${eXo.env.portal.context}/${eXo.env.portal.rest}/gamification/realizations/api/getExport?fromDate=${fromDate}&toDate=${toDate}`, '_blank');
+export function exportFile(fromDate, toDate, earnerId, returnType) {
+  window.open(`${eXo.env.portal.context}/${eXo.env.portal.rest}/gamification/realizations/api/allRealizations?fromDate=${fromDate}&toDate=${toDate}&earnerId=${earnerId}&returnType=${returnType}`, '_blank');
 }
```

Comment:
no need to add a parameter if the returnType is always `xls`
```suggestion
export function exportFile(fromDate, toDate, earnerId) {
  window.open(`${eXo.env.portal.context}/${eXo.env.portal.rest}/gamification/realizations/api/allRealizations?fromDate=${fromDate}&toDate=${toDate}&earnerId=${earnerId}&returnType=xls`, '_blank');
}
```